.github/workflows/release.yml

```diff
@@ -41,35 +41,39 @@ jobs:
             platform: win32
             arch: x64
             regular_build: 'true'
+            cross: false
           - os: windows-latest
             rust-target: aarch64-pc-windows-msvc
             platform: win32
             arch: arm64
+            cross: false
           - os: ubuntu-20.04
-            rust-target: x86_64-unknown-linux-gnu
+            rust-target: x86_64-unknown-linux-musl
             platform: linux
             arch: x64
             regular_build: 'true'
             cross: true
           - os: ubuntu-20.04
-            rust-target: aarch64-unknown-linux-gnu
+            rust-target: aarch64-unknown-linux-musl
             platform: linux
             arch: arm64
             cross: true
           - os: ubuntu-20.04
-            rust-target: arm-unknown-linux-gnueabihf
```

Comment:
not sure about this. i dont know the difference between arm targets

Reply:
We ran into some errors in https://github.com/Myriad-Dreamin/tinymist/pull/884. I guessed we should change targets supported by cross after diff typst CI scripting with ours.